src/Illuminate/Filesystem/FilesystemAdapter.php

```diff
@@ -328,13 +329,25 @@ public function url($path)
      *
      * @param  \League\Flysystem\AwsS3v3\AwsS3Adapter  $adapter
      * @param  string  $path
+     * @param  string  $expires
      * @return string
      */
-    protected function getAwsUrl($adapter, $path)
+    protected function getAwsUrl($adapter, $path, $expires)
```

Comment:
This is not optional, therefore a breaking change for whoever overrides it. Should target 5.5.